content/en/docs/collector/custom-collector.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Building your own Collector instance
```

Comment:
I'd suggest the following title since it more closely matches the other pages that were recently added:

# Building a custom collector

Reply:
I am good with that

Reply:
Good. Updated.